src/typings/notifications.ts

```diff
@@ -0,0 +1,32 @@
+import { EntryReviewType } from '#typings';
+
+export type NotificationStatus = 'seen' | 'unseen';
+export type NotificationType = 'project_join_request'
+    | 'project_join_response'
+    | 'project_join_request_abort'
+    | 'entry_review_comment_add'
+    | 'entry_review_comment_modify'
+
+export interface NotificationFields {
+    id: number;
+    project: number;
+    receiver: 5;
```

Comment:
Sir what?
Is receiver only allowed to be 5?

Reply:
naah, should be a `number` or `UserFields['id']`